parlai/agents/examples/seq2seq.py

```diff
@@ -48,10 +48,12 @@ def __init__(self, embeddings, hidden_size):
         # must call super on all nn.Modules.
         super().__init__()
 
-        _vocab_size, esz = embeddings.weight.shape
         self.embeddings = embeddings
         self.lstm = nn.LSTM(
-            input_size=esz, hidden_size=hidden_size, num_layers=1, batch_first=True
+            input_size=hidden_size,
```

Comment:
Yes, otherwise we need an extra layer to project from hidden-size to embedding-size for the final softmax. As this is just a tutorial, this sort of simplification is fine imho.

Reply:
sounds good